arcs/logparser.py

```diff
@@ -40,28 +40,35 @@
 def _convert_type(k, v):
     return TYPE_CONVERTERS.get(k, lambda x: x)(v)
 
+
```

Comment:
I'm still not seeing all the same pep8 stuff as you. Need to figure out what's going on there.